src/wacks4shop/wack.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import logging
+from datetime import datetime
+from pathlib import Path
+
+from dotenv import load_dotenv
+
+from wacks4shop.shift4shop import Shift4Shop
+from wacksbywarby.constants import WACK_ERROR_SENTINEL
+from wacksbywarby.db import Wackabase
+from wacksbywarby.discord import Discord
+from wacksbywarby.wack import announce_new_sales, get_inventory_state_diff
+
+DATABASE_DIR = "data/wacks4shop"
+
+load_dotenv()
+
+logger = logging.getLogger("wacks4shop")
+
+
+def main(db: Wackabase, dry=False):
+    try:
+        logger.info("TIME TO WACK")
+        logger.info("Dry run: %s", dry)
+
+        discord = Discord(debug=dry)
+        shift4shop = Shift4Shop(debug=dry)
+
+        now = datetime.now()
+
+        last_timestamp = db.get_timestamp()
+        sales = shift4shop.determine_sales(timestamp=last_timestamp)
+
+        if not sales:
+            return
+
+        # grab the current number of total sales
+        previous_num_sales = db.get_last_num_sales()
+        current_num_sales = shift4shop.get_num_sales()
+        logger.info(
+            f"current num sales: {current_num_sales}, previously stored num sales: {previous_num_sales}"
+        )
+        # handle the case where the shop owner manually lowers their own inventory
+        # instead of inventory lowering coming from a sale
+        if not current_num_sales > previous_num_sales:
+            logger.info(
+                "num sales did not increase, skipping announcement (%d --> %d)",
+                current_num_sales,
+                previous_num_sales,
+            )
+        else:
+            announce_new_sales(discord, sales, current_num_sales)
```

Comment:
Can this logic be simplified to 
```
previous_num_sales = db.get_last_num_sales()
current_num_sales = previous_num_sales + len(sales)
if len(sales):
    announce_new_sales(discord, sales, current_num_sales)
else:
     logger.info(
                "num sales did not increase, skipping announcement (%d --> %d)",
                current_num_sales,
                previous_num_sales,
            )
```

Reply:
mm yeah I guess we don't have to worry about the case where the store owner manually lowers inventory anymore, since it won't count as an order, hooray!

Reply:
fixed! https://github.com/allisonking/wacks-by-warby/pull/7/commits/4759f6b89730bd8ffb6c32cba333ed3bf59ec016